docs/examples/ratify-high-availability.md

```diff
@@ -0,0 +1,141 @@
+# Install Ratify for High Availability
+
+The default Ratify installation relies on a single Ratify pod processing all requests. For higher performance and availability requirements, Ratify can be set to run with multiple replicas and a shared state store.
+
+## Add Helm Chart Dependencies
+```bash
+helm repo add dapr https://dapr.github.io/helm-charts/
+helm repo add bitnami https://charts.bitnami.com/bitnami
+helm repo add gatekeeper https://open-policy-agent.github.io/gatekeeper/charts
+helm repo add ratify https://deislabs.github.io/ratify
+helm repo update
+```
+
+## Install Dapr
+```bash
+helm upgrade --install dapr dapr/dapr --namespace dapr-system --create-namespace --wait
+```
+
+## Install Gatekeeper
+```bash
+helm install gatekeeper/gatekeeper  \
+    --name-template=gatekeeper \
+    --namespace gatekeeper-system --create-namespace \
+    --set enableExternalData=true \
+    --set validatingWebhookTimeoutSeconds=5 \
+    --set mutatingWebhookTimeoutSeconds=2
+```
+
+## Install Redis
```

Comment:
Does Redis use memory as the persistent storage by default? How to configure a CSI as a backend storage interface?

Reply:
Yes the default redis chart sets up a persistent storage by default